h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -188,6 +210,17 @@ export default function AnnotationCard({
                 classes="text-color-text hyp-wrap-anywhere"
               />
             </Excerpt>
+            {annoBodyCollapsible && (
+              <div className="flex justify-end !mt-1">
+                <Button
+                  onClick={() => setAnnoBodyCollapsed(prev => !prev)}
+                  data-testid="toggle-anno-body"
+                >
+                  {annoBodyCollapsed ? <ExpandIcon /> : <CollapseIcon />}
+                  Show {annoBodyCollapsed ? 'more' : 'less'}
+                </Button>
```

Comment:
I notice this button is a little larger than the one in the client. In the client it is 34px high on my system, vs 38 here.

Reply:
Yeah, there's no explicit classes or props that could affect the size being set in the client. It's just apparently inheriting a 18px line-height, while it is 22px in h.